client/me/help/main.jsx

```diff
@@ -127,29 +151,39 @@ const Help = React.createClass( {
 	render: function() {
 		const {
 			isEmailVerified,
-			showCoursesTeaser
+			userId
 		} = this.props;
 		return (
 			<Main className="help">
 				<MeSidebarNavigation />
 				<HelpSearch />
 				{ ! isEmailVerified && <HelpUnverifiedWarning /> }
-				{ showCoursesTeaser && this.getCoursesTeaser() }
+				{ this.getCoursesTeaser() }
```

Comment:
Why the change here? Previously, if `showCoursesTeaser` was `false` the loading placeholder wouldn't show, which seems a bit less jarring if it doesn't eventually load something?